### ISSUE TITLE
Getting logs from a successful installation screen

### ISSUE THREAD

**Issue by lkocman** [agama-project/agama#1447]
Hello team,

we're debugging issue #1446 where installation passes and system is not bootable and it would still be nice to be able to get logs even from a successful installation.  I already got used so much to convenience to fetch logs from a webbroswer.

![image](https://github.com/openSUSE/agama/assets/510119/4c802bee-4bbb-404d-8a23-6416fd46bee2)

```
Current steps is

agama:~ # tar -cvf YaST2.tar /var/log/YaST2 # no y2save_logs
tar: Removing leading `/' from member names
/var/log/YaST2/
/var/log/YaST2/storage-inst/
/var/log/YaST2/storage-inst/01-probed.xml
/var/log/YaST2/storage-inst/02-proposed.yml
/var/log/YaST2/storage-inst/03-committed.xml
/var/log/YaST2/storage-inst/03-committed.yml
/var/log/YaST2/storage-inst/02-proposed.xml
/var/log/YaST2/storage-inst/01-probed.yml
/var/log/YaST2/control-inst/
/var/log/YaST2/control-inst/control.xml
/var/log/YaST2/control-inst/README
/var/log/YaST2/y2log-1.gz
/var/log/YaST2/y2log
/var/log/YaST2/config_diff_2024_07_07.log

agama:~ # vim ~/.ssh/authorized_keys # no way I'm retyping that password

lkocman@localhost:~> sftp root@192.168.100.135
Connected to 192.168.100.135.
sftp> ls
YaST2.tar  bin        
sftp> get YaST2.tar 
Fetching /root/YaST2.tar to YaST2.tar
YaST2.tar                                                                                   100% 2380KB 380.8MB/s   00:00    
sftp> 


```

**Comment by lkocman** [comment]
logs from agama-installer-openSUSE.x86_64-9.0.0-openSUSE-Build5.3.iso (downloaded today in the morning)
[YaST2.zip](https://github.com/user-attachments/files/16125184/YaST2.zip)

**Comment by lkocman** [comment]
Fixed by https://build.opensuse.org/package/rdiff/openSUSE:Leap:16.0/000release-packages?linkrev=base&rev=25

![image](https://github.com/openSUSE/agama/assets/510119/019b0c63-218b-45dd-94fb-0e96f9882ffa)

**Comment by ancorgs** [comment]
> Fixed by https://build.opensuse.org/package/rdiff/openSUSE:Leap:16.0/000release-packages?linkrev=base&rev=25

I think you posted this in the wrong issue. I guess you meant #1446

**Comment by stacheldrahtje** [comment]
Weird, since he posted this yesterday I surmised it was new.

**Comment by dgdavid** [comment]
@lkocman this is now fixed by #1690 :tada:
